packages/plugins/integration-zapier/zapier/src/triggers/taskList.ts

```
@@ -0,0 +1,60 @@
+            url: `${process.env.API_BASE_URL}/api/tasks`,
+            method: 'GET',
+            headers: {
+                Authorization: `Bearer ${bundle.authData['access_token']}`,
+            },
+            params: {
+                projectId: projectId
+            }
+        });
+        
+        // Check if response data is in expected format
+        if (response.data && response.data.items) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)